exercises/input-output/src/main/java/com/cbfacademy/WriteStream.java

```diff
@@ -0,0 +1,35 @@
+package com.cbfacademy;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class WriteStream {
+    public void writeStream() throws IOException{
```

Comment:
you shouldn't need to declare this `throws` here, you're catching exceptions within your method